copier/errors.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, features: Sequence[str]):
         )
 
 
+class YieldTagInFileError(CopierError):
+    """A yield tag is used in the file content, but it is not allowed."""
+
+    pass
```

Comment:
`pass` is unnecessary if there's a docstring :slightly_smiling_face: 

```suggestion
```

Reply:
thanks! I fixed in [6f4acec](https://github.com/copier-org/copier/pull/1855/commits/6f4acecccb67d0f2e896d81ec0580eb499651dbb) .